<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>921</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Directory Options</string>
   </property>
   <widget class="QPushButton" name="btnEncryptSearch">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search to Encrypt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>901</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecryptSearch">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search to Decrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>921</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>901</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptDecryptButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt/Decrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded. In mode Encrypt</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Status: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
